sensor_msgs/Image depth
sensor_msgs/Image marker
geometry_msgs/Pose Twc
std_msgs/String cam_id
float32 fx
float32 fy
---
visualization_msgs/MarkerArray output
